index.html

```diff
@@ -61,6 +61,260 @@ <h1 class="name">
+        you see and have a project you need coded, don't hestiate to contact me..
+      </p>
+
+      <h4 class="connect">LETS CONNECT</h4>
+
+      <div class="socialmedia_icons_container">
+        <ul class="media_icons">
+          <li>
+            <a href="#">
+              <img src="images/icons/cat.svg" alt="Github" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="media--images"
+                src="images/icons/linkedin.png"
+                alt="LinkedIn"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/fingers.svg" alt="Victory" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+              class="media--images"
+              src="images/icons/twitter.svg"
+              alt="Twitter"
+            />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+              class="media--images"
+              src="images/icons/m.svg"
+              alt="Media Icon"
+            />
+            </a>
+          </li>
+        </ul>
+        <div class="btn-container about-btn">
+          <button type="button" class="btn-xxl btn-resume">
+            Get my resume
+          </button>
+        </div>
+      </div>
+
+      <ul class="accordion">
+        <li>
+          <div class="accordion-container">
+            <a class="skill--description-link" href="#"
+              ><span class="skill--description">Languages</span></a
+            >
+            <ion-icon
+              class="chevron-icons"
+              name="chevron-down-sharp"
+            ></ion-icon>
+          </div>
+          <ul>
+            <li class="skill">
+              <img class="skill-image" src="images/works/skill_1.png" alt="" />
+              <span class="skill-name">Javascript</span>
+            </li>
+            <li class="skill">
+              <img class="skill-image" src="images/works/skill_2.png" alt="" />
+              <span class="skill-name">HTML</span>
+            </li>
+            <li class="skill">
+              <img class="skill-image" src="images/works/skill_3.png" alt="" />
+              <span class="skill-name">CSS</span>
+            </li>
+          </ul>
+        </li>
+      </ul>
+
+      <div class="frameworks-container">
+        <a class="framework--title-link" href="#">
+          <span class="framework--title">Frameworks</span></a
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
+      <hr />
+      <div class="skills-container">
+        <a class="skills--title-link" href="#">
+          <span class="skills--title">Skills</span></a
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
```

Comment:
- [ ] You can wrap the three main technologies sections with a `ul` tag as well. Kindly do so. You can use something like the following:

```javascript
 <ul>
    <li>
        <>Languages</>
        <ul>
            <li>Javascript</li>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
    <li>
        <>Frameworks<>
        <ul>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
    <li>
        <>Skills</>
        <ul>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
</ul>
```